webpack/assets/javascripts/react_app/components/AuthSource/Welcome.js

```diff
@@ -15,10 +15,10 @@ export const WelcomeAuthSource = ({ canCreate }) => {
       <FormattedMessage
         id="LDAP-providers"
         defaultMessage={__(
-          'The authentication process currently requires an LDAP provider, such as {FreeIPA}, {OpenLDAP} or {Microsoft}.'
+          'The authentication process currently requires an LDAP provider, such as {Free_IPA}, {OpenLDAP} or {Microsoft}.'
         )}
         values={{
-          FreeIPA: <em>FreeIPA</em>,
+          Free_IPA: <em>FreeIPA</em>,
```

Comment:
The formatted message is not broken anymore, but the branding for **FreeIPA** is missing.
![image](https://github.com/user-attachments/assets/f89fc612-ed46-41cd-9055-f1941813206d)

```suggestion
          Free_IPA: <em>{__('FreeIPA')}</em>,
```

This way, **FreeIPA** is still branded to **Red Hat Identity Management**.
![image](https://github.com/user-attachments/assets/bd190310-982c-427a-9ba2-bd6729f7cee3)

I don't think **OpenLDAP** or **Active Directory** need the same treatment, as they do not need to be branded.

Reply:
Thanks, fixed!